rules-reviewed/eap7/eap6/resteasy.windup.xml

```diff
@@ -0,0 +1,826 @@
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="JAX-RS API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ext/ReaderInterceptor.html" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000015"> <!-- Not clear what it replace!! -->
+            <when>
+                <javaclass references="org.jboss.resteasy.core.interception.InterceptorRegistryListener" />
+            </when>
+            <perform>
+                <hint title="Deprecated InterceptorRegistryListener interface" effort="1" severity="mandatory">
+                    <message>Use ### instead of `org.jboss.resteasy.core.interception.InterceptorRegistryListener`.
+                    The Resteasy interceptor facility introduced in release 2.x is replaced by the JAX-RS 2.0 compliant interceptor facility in release 3.0.x.</message>
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="JAX-RS API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ext/ReaderInterceptor.html" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000016"><!-- Not clear what it replace!! -->
+            <when>
+                <javaclass references="org.jboss.resteasy.core.interception.ClientExecutionContextImpl" />
+            </when>
+            <perform>
+                <hint title="Deprecated ClientExecutionContextImpl class" effort="1" severity="mandatory">
+                    <message>Use ### instead of `org.jboss.resteasy.core.interception.ClientExecutionContextImpl`.
+                    The Resteasy interceptor facility introduced in release 2.x is replaced by the JAX-RS 2.0 compliant interceptor facility in release 3.0.x.</message>                    
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="JAX-RS API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ext/ReaderInterceptor.html" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000017">
+            <when>
+                <javaclass references="org.jboss.resteasy.client.core.executors.ApacheHttpClientExecutor" />
```

Comment:
For some reason, this class has never been shipped with EAP 6, even though it exists in upstream repositories of the corresponding versions. Oh well :-(